src/routes/posts/+page.server.ts

```diff
@@ -0,0 +1,46 @@
+import type { PageServerLoad } from './$types';
+import type { Article } from '$lib/types';
+
+export const load: PageServerLoad = async ({ platform, url }) => {
+	const page = parseInt(url.searchParams.get('page') ?? '0');
+
+	const response = await platform?.env.DB.prepare(
+		`SELECT article_ids FROM date_entries 
+		WHERE LENGTH(article_ids) > 0
+		ORDER BY date DESC LIMIT ${page * 5}, 5`
+	).all();
+
+	const ids =
+		response?.results
+			.map((resp: Array<{ article_ids: string }> | unknown) =>
+				(resp as { article_ids: string }).article_ids.split(',')
+			)
+			.flat()
+			.filter((id) => id) ?? [];
```

Comment:
What case does this `filter` handle?

Reply:
real, I was trying something else and didn't remove this